src/main/java/com/strandls/dataTable/pojo/DataTableListMapping.java

```diff
@@ -0,0 +1,110 @@
+package com.strandls.dataTable.pojo;
+
+import java.util.Date;
+
+import com.strandls.user.pojo.UserIbp;
+
+public class DataTableListMapping {
+
+	
+	private Long id;
```

Comment:
format your code and add a default constructor

Reply:
added default constructor